docs/sphinx/reference-properties.rst

```diff
@@ -690,3 +721,14 @@ Property Modification Functions
 ---------------------
 
 .. function:: void obs_property_button_set_url(obs_property_t *p, char *url)
+
+---------------------
+
+.. function:: void obs_property_info_set_type(obs_property_t *p, enum obs_info_type type)
+
+   :param   type: Can be one of the following values:
+
+                  - **OBS_INFO_NORMAL** - To show an info
+                  - **OBS_INFO_WARNING** - To show a warning
+                  - **OBS_INFO_ERROR** - To show an error
+   :return:       The property
```

Comment:
void function has no return

Reply:
I discovered that I copied and pasted en already existing error in the doc `obs_property_button_set_type` has the same error.